tools/rstlint.py

```diff
@@ -221,8 +312,10 @@ def main(argv):
     else:
         for severity in sorted(count):
             number = count[severity]
-            print('%d problem%s with severity %d found.' %
-                  (number, number > 1 and 's' or '', severity))
+            print(
+                '%d problem%s with severity %d found.'
+                % (number, number > 1 and 's' or '', severity)
```

Comment:
I would rewrite `number > 1 and 's' or ''` as `'s' if number > 1 else ''`.

Reply:
I agree 👍